drivers/dma/dma_sam0.c

```diff
@@ -149,7 +149,7 @@ static int dma_sam0_config(struct device *dev, u32_t channel,
 				     DMAC_CHCTRLA_TRIGSRC(config->dma_slot);
 	} else {
 		/* One peripheral trigger per beat */
-		chcfg->CHCTRLA.reg = DMAC_CHCTRLA_TRIGACT_BURST |
+		chcfg->CHCTRLA.reg = DMAC_CHCTRLA_TRIGACT_BLOCK |
```

Comment:
The burst length setting code below should also be removed, since it won't do anything if not using burst mode (which apparently doesn't work, oh well).  That is, all the code below related to burst length can also be removed, since the defaults (0) are fine in block mode.

Reply:
OK, removed.

This makes the code even more similar between the two versions of the peripheral.